webclient/nginx.conf

```diff
@@ -123,13 +123,12 @@ http {
       try_files /index-view-view-$THEME-$LANG.html /404.html;
     }
 
-    location ^~ /assets/ {
-      expires 1d;
+    location ~ ^/(js|assets)/cache_.*$ {
+      expires 30d;
```

Comment:
We may want to increase this value in the future, but for now this seems fine

Reply:
Nginx has the option `expires max` iirc, but yeah, we can change that in the future.